### PR TITLE
[release-v1.10] Add eventshub service port name

### PR DESCRIPTION
**Pull request by pierDipi**
Same as https://github.com/openshift-knative/eventing/pull/289

### CODE REVIEW

File: openshift/release/artifacts/eventing-kafka-controller.yaml
Comment (pierDipi) [inline openshift/release/artifacts/eventing-kafka-controller.yaml:1]: These are generated changes

File: openshift/release/generate-release.sh [hunk 1]
```diff
@@ -7,12 +7,14 @@ source $(dirname $0)/resolve.sh
 GITHUB_ACTIONS=true $(dirname $0)/../../hack/update-codegen.sh
 git apply openshift/patches/disable-ko-publish-rekt.patch
 git apply openshift/patches/override-min-version.patch
+git apply openshift/patches/add-eventshub-port-name.patch
+git apply openshift/patches/unique-secret-names.patch
```
Comment (ReToCode) [inline openshift/release/generate-release.sh:11]: Not sure if this should be here or not? If ok, feel free to unhold.

**Review by ReToCode — APPROVED** [review]
/lgtm
/approve
/hold (see comment)

File: openshift/release/generate-release.sh (hunk 1 above)
Comment (pierDipi) [reply openshift/release/generate-release.sh:11]: Yes, that's what missing, without this `make generated-files` will revert that patch

**Pull request merged**